### PR TITLE
TM-20: Add organizer landing page

### PR DESCRIPTION
**Pull request by gorkaj**
For now it's a page with hardcoded mock data &rarr; it will be fixed after retrieving currently logged user

### CODE REVIEW

**Review by bjadczak — CHANGES_REQUESTED** [review]
Please fix minor issues

File: src/front/ticketer/lib/auth/widget_tree.dart [hunk 1]
```diff
@@ -1,8 +1,12 @@
 import 'package:ticketer/auth/auth.dart';
-import 'package:ticketer/pages/page_login.dart';
+import 'package:ticketer/auth/user.dart';
+import 'package:ticketer/model/user_type.dart';
+import 'package:ticketer/pages/organizer/organizer_landing_page.dart';
 import 'package:ticketer/pages/page_home.dart';
 import 'package:flutter/material.dart';
 
+import '../pages/login/page_login.dart';
```
[the remark this answers is not in the document]
Comment (gorkaj) [reply src/front/ticketer/lib/auth/widget_tree.dart:5]: done

[the remark this answers is not in the document]
Comment (bjadczak) [reply src/front/ticketer/lib/auth/widget_tree.dart:5]: No it is not done. :(

**Review by bjadczak — CHANGES_REQUESTED** [review]

**Review by bjadczak — APPROVED** [review]

File: src/front/ticketer/lib/auth/widget_tree.dart (hunk 1 above)
[the remark this answers is not in the document]
Comment (gorkaj) [reply src/front/ticketer/lib/auth/widget_tree.dart]: done:d

**Review by faliszewskii — APPROVED** [review]
LGTM

**Pull request merged**